src/App/ViewModels/SearchViewModel.cs

```diff
@@ -0,0 +1,35 @@
+﻿using App.Models;
+using Flurl;
+using Flurl.Http;
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.ComponentModel;
+using System.Text;
+using System.Threading.Tasks;
+using Xamarin.Forms;
+
+namespace App.ViewModels
+{
+    class SearchViewModel : INotifyPropertyChanged
+    {
+        public ObservableCollection<Jobs> Jobss { get; set; }
+        public Command LlenarListaCommand { get; set; }
```

Comment:
Coding should be done in English.

Reply:
Removida las propiedades que no se utilizan